plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/stock-indicator/test/block.test.tsx

```diff
@@ -58,6 +58,10 @@ jest.mock( '@woocommerce/settings', () => ( {
 				simple: 'Simple product',
 			};
 		}
+		if ( param === 'globalPaymentMethods' ) {
```

Comment:
`assets/js/settings/blocks/constants.ts` is loaded as part of this test (via a long stacktrace) now that we are importing constants to use in the `validateAdditionalFields` function. This test mocks `getSetting`, but it doesn't mock some of the keys used in the constants file, so the test fails. Added these extra mocks here